### ISSUE TITLE
 [MAS1.4.3] Luminosity Contrast Ratio of 'Mic' button icons is less than required.

### ISSUE THREAD

**Issue by priysn** [Microsoft/BotFramework-WebChat#1411]
Actual Result:
Luminosity Contrast Ratio of 'Send' button icon is 2.4:1.

Expected Result:
Luminosity Contrast Ratio of 'Send' button should be greater than or equal to 4.5:1.

Repro Steps:

1. Open URL [https://apac01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fmicrosoft.github.io%2FBotFramework-WebChat%2Fspeech-cognitive-services-bing-speech%2F%3Fs%3D7c7dacfe04044f4f99a1edcf9b0e2ab9&data=02%7C01%7Cv-saugup%40microsoft.com%7C0f6f5e63167449daa46d08d65570f676%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636790341026155420&sdata=gcFq8JsflCLfUKAyw8f2A19aJcEQEFxGzjASMK9S59A%3D&reserved=0](https://apac01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fmicrosoft.github.io%2FBotFramework-WebChat%2Fspeech-cognitive-services-bing-speech%2F%3Fs%3D7c7dacfe04044f4f99a1edcf9b0e2ab9&data=02%7C01%7Cv-saugup%40microsoft.com%7C0f6f5e63167449daa46d08d65570f676%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636790341026155420&sdata=gcFq8JsflCLfUKAyw8f2A19aJcEQEFxGzjASMK9S59A%3D&reserved=0) in Edge browser.
2. Notice 'Upload File' and 'Mic' button icons present in bottom of screen.
3. Open 'Colour Contrast Analyser' and select Foreground & Background color of mentioned elements one by one.
4. Notice value of Luminosity Contrast Ratio for Mic button.

User Impact:
Users with low vision or Color blindness will find it difficult to access content with low luminosity contrast ratio.

**Test Environment:**
**OS:** Windows 10
**OS Build:** 17134.345
**OS Version:** 1803
**Browser:** Edge
**Tool:** Color Contrast Analyser

**Attachment:**
![mas1 4 3_cca for mic button](https://user-images.githubusercontent.com/41668935/49214625-192d5480-f3ed-11e8-9faf-09f172e9acf2.png)

**Comment by Amit8527510735** [comment]
#HCL;#Accessibility;#HCL_BotFramework_WebChat;A11yMAS;WCAG1.4.3
